apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -486,6 +509,7 @@ public SpannerConnectionStringBuilder(string connectionString, ChannelCredential
         /// Must not be null.</param>
         /// <param name="credentials">The credential to use for the connection. May be null.</param>
         /// <param name="sessionPoolManager">The session pool manager to use. Must not be null.</param>
+        [Obsolete("Use overloads accepting credentials, then set the SessionPoolManager property", error: false)]
```

Comment:
We usually don't set `error: false` on obsolete attributes. (But I'm fine either way, just mentioning for consistency).

Reply:
I figured it would just be a little clearer - I can definitely remove it if we want.